components/eventing-controller/config/crd/bases/eventing.kyma-project.io_eventingbackends.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
+    controller-gen.kubebuilder.io/version: v0.2.5
```

Comment:
Let's upgrade to https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.6.0 if you are willing to.

Then you need to change the [Makefile](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/Makefile#L136) as well.

Finally, execute the `make copy-crds` target to sync CRDs in the resources folder.